### PR TITLE
Enterprise-traces docs: Fixed broken links

### PR DESCRIPTION
**Pull request by xogoodnow**
Fixed a couple of broken links.

**Special notes for your reviewer:**
Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

**Comment by irenerl24** [comment]
Hi @xogoodnow!  

Thank you so much for your PR! I see you're trying to use the Github source files, and we currently do not have plans to make links also work in the GitHub rendered Markdown.

However, thanks to your PR, I've noticed that some links do not follow [our convention](https://grafana.com/docs/writers-toolkit/write/links/), so if you don't mind, I'll update the links using version substitution in this PR.

File: docs/sources/datasources/tempo/query-editor/_index.md [hunk 1]
```diff
@@ -123,15 +123,15 @@ Using the service graph view, you can:
 - Determine how long the slowest queries in your service take to complete.
 - Examine all traces that contain spans of particular interest based on rate, error, and duration values (RED signals).
 
-For more information about the service graph, refer to [Service graph](../service-graph/).
+For more information about the service graph, refer to [Service graph](../service-graph.md).
```
Comment (irenerl24) [inline docs/sources/datasources/tempo/query-editor/_index.md]: For more information about the service graph, refer to [Service graph](../service-graph/)

**Review by irenerl24 — APPROVED** [review]

File: docs/sources/datasources/tempo/query-editor/_index.md (hunk 1 above)
Comment (xogoodnow) [reply docs/sources/datasources/tempo/query-editor/_index.md]: Hi, @irenerl24,

Oh, I did not know about the links.
Thank you .

Cheers

**Comment by xogoodnow** [comment]
> Hi @xogoodnow!
> 
> Thank you so much for your PR! I see you're trying to use the Github source files, and we currently do not have plans to make links also work in the GitHub rendered Markdown.
> 
> However, thanks to your PR, I've noticed that some links do not follow [our convention](https://grafana.com/docs/writers-toolkit/write/links/), so if you don't mind, I'll update the links using version substitution in this PR.

hI @irenerl24,

Sure, I did not know about the convention.
If there is anything that I must do, please let me know.
Thank you.


Cheers

**Pull request merged**